### PR TITLE
Adding the collab space for the ecosystem report

### PR DESCRIPTION
**Pull request by edsadr**
This PR address is [#1253] (https://github.com/openjs-foundation/cross-project-council/issues/1253#issuecomment-2007943588)

It will require the creation of the repo at https://github.com/openjs-foundation/ecosystem-report @bensternthal  your help will be needed for this.

The initial members of the space, according to the discussion on the issue, are:

- @anfibiacreativa
- @PaulaPaul 
- @tobie 
- @mhdawson 
- @italojs

### CODE REVIEW

**Review by PaulaPaul — APPROVED** [review]
Thank you @edsadr - lgtm

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by bensternthal** [comment]
Repo created and the specified folks have been added https://github.com/openjs-foundation/ecosystem-report

Have fun!!

**Review by ljharb — APPROVED** [review]

**Review by VoltrexKeyva — APPROVED** [review]

**Review by joesepi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -86,11 +86,13 @@ If you are interested in fostering collaboration with the support of the OpenJS
 
 | Name                                         |  Repository                        | Description                                                     |
 | -------------------------------------------- |:----------------------------------:|-----------------------------------------------------------------|
+| Ecosystem Report                             | [Repository][ecosystem-report]     | JavaScript ecosystem anual end-users poll and report            |
```
Comment (ctcpip) [inline README.md:89]:
```suggestion
| Ecosystem Report                             | [Repository][ecosystem-report]     | JavaScript ecosystem annual end-users poll and report            |
```

Comment (ctcpip) [reply README.md:89]: https://github.com/openjs-foundation/cross-project-council/pull/1288

**Comment by anfibiacreativa** [comment]
@bensternthal could you please reissue the invitation? I was on vacation and got to late to it.

**Pull request merged**